schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
-
-CREATE TABLE animals (
-    name varchar(100)
-);
+CREATE TABLE animals(
+    id INT GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR(50),
+    date_of_birth DATE,
+    escape_attempts INT,
+    neutered BIT,
```

Comment:
- [ ] According to project requirements the column `neutered ` is `boolean` not `BIT`. a bit type only stores the numbers 0 and 1 (or null). note that the boolean only stores true and false (or null). A number (0, 1) is not a boolean. A boolean value can be used anywhere a boolean expression is expected. Please fix it :+1:

Reply:
Thank you for this correction. The change has been updated